src/card.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WITCH OF MOUNTAIN WINDS</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+    <link 
+      rel="shortcut icon" 
+      href="./images/svg/logo-colour.svg" 
+      type="image/png"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap" rel="stylesheet">
```

Comment:
The task requires using Sharp Sans No2 Medium and Avenir fonts, but you have imported Poppins font.